helm-chart/renku-core/values.yaml

```diff
@@ -15,9 +15,9 @@ cleanupProjectsTTL: 1800
 logLevel: WARNING
 
 image:
-  repository: renku/renku-core
+  repository: psam/renku-core
```

Comment:
```suggestion
  repository: renku/renku-core
```

Reply:
I know - ty! I'm waiting for 1052 and 1048 to be merge so I can test this PR one last time before merging it.